go-controller/pkg/ovn/master.go

```diff
@@ -108,8 +108,8 @@ func (oc *Controller) Start(nodeName string, wg *sync.WaitGroup, ctx context.Con
 				// we need to handle the transition properly like clearing
 				// the cache. It is better to exit for now.
 				// kube will restart and this will become a follower.
-				klog.Infof("No longer leader; exiting")
-				os.Exit(1)
+				klog.Infof("No longer leader;")
```

Comment:
@jluhrsen I think we should either keep the whole "; exiting" or drop the trailing ";"

Reply:
yes. fixed.